src/drawer.canvasentry.js

```diff
@@ -86,10 +86,7 @@ export default class CanvasEntry {
      */
     initProgress(element) {
         this.progress = element;
-        this.progressCtx = this.progress.getContext(
-            '2d',
-            this.canvasContextAttributes
-        );
+        this.progressCtx = this.progress.getContext('2d', this.canvasContextAttributes);
```

Comment:
no need for this formatting only change.

Reply:
Like I said, feel free to ignore the parts that don't apply. I just wanted to throw this solution out there; I'm not expecting this PR to make it to master as-is